app/react/components/message.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { PropTypes } from 'react';
+import moment from 'moment';
+
+const Message = (props) => {
+  return(
+    <div className="entry list-group-item">
+      <span><strong>{props.userName} </strong> </span>
+      <span>- {moment(props.createdAt).calendar()}</span>
+      <br/>
+      <span>{props.content}</span>
+    </div>
+  )
+}
```

Comment:
some tricks you could use to make this component a bit more terse: implicit return + destructure arguments. Like this:

```jsx
const Message = ({ userName, createdAt, content }) => (
  <div className="entry list-group-item">
    <span><strong>{userName} </strong> </span>
    <span>- {moment(createdAt).calendar()}</span>     
    <br/>
    <span>{content}</span>
  </div>
);
```

of course it's completely optional ;)